conference/user_panel.py

```diff
@@ -238,6 +250,234 @@ def __init__(self, *args, **kwargs):
+    company = AddSpeakerToTalkForm.base_fields["company"]
+    company_homepage = AddSpeakerToTalkForm.base_fields["company_homepage"]
+
+    bio = forms.CharField(
+        label="Compact biography",
+        help_text="Short biography (one or two paragraphs). Do not paste your CV",
+        widget=forms.Textarea,
+        required=False,
+    )
+    tagline = forms.CharField(
+        label="Tagline", help_text="Describe yourself in one line.", required=False
+    )
+    twitter = forms.CharField(max_length=80, required=False)
+    visibility = forms.ChoiceField(
+        label="",
+        choices=ATTENDEEPROFILE_VISIBILITY,
+        widget=forms.RadioSelect,
+        required=False,
+    )
+
+    # The following fields are rendered manually, not using crispy forms, in
+    # order to have more control over their layout.
+    picture_options = forms.ChoiceField(
+        label="",
+        choices=(
+            ("none", "Do not show any picture"),
+            ("gravatar", "Use my Gravatar"),
+            ("url", "Use this url"),
+            ("file", "Use this picture"),
+        ),
+        required=False,
+        widget=forms.RadioSelect,
+    )
+    image_url = forms.URLField(required=False)
+    image = forms.FileField(required=False, widget=forms.FileInput)
+
+    class Meta:
+        model = AttendeeProfile
+        fields = (
+            # first section
+            "first_name",
+            "last_name",
+            "is_minor",
+            "phone",
+            "email",
+            # second section
+            "picture_options",
+            "image_url",
+            "image",
+            # third section
+            "tagline",
+            "twitter",
+            "personal_homepage",
+            "location",
+            "job_title",
+            "company",
+            "company_homepage",
+            "bio",
+            # fourth section
+            "visibility",
+        )
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        # Set the initial values for fields that are not part of AttendeeProfile
+        user = self.instance.user
+        self.fields["first_name"].initial = user.first_name
+        self.fields["last_name"].initial = user.last_name
+        self.fields["email"].initial = user.email
+
+        p3_profile = self.instance.p3_profile
+        self.fields["tagline"].initial = p3_profile.tagline
+        self.fields["twitter"].initial = p3_profile.twitter
+
+        self.fields["bio"].initial = getattr(self.instance.getBio(), "body", "")
+
+        # Determine the value of the image fields
+        image_url = self.instance.p3_profile.profile_image_url()
+        if self.instance.image:
+            selected_image_option = "file"
+        elif p3_profile.image_url:
+            selected_image_option = "url"
+        elif p3_profile.image_gravatar:
+            selected_image_option = "gravatar"
+        else:
+            selected_image_option = "none"
```

Comment:
I'd use some enum for those values. Maybe just define the choices above as `Choices`